classes/pdf/HTMLTemplateInvoice.php

```diff
@@ -291,18 +291,18 @@ public function getContent()
          */
         $round_type = null;
         switch ($this->order->round_type) {
-        case Order::ROUND_TOTAL:
+            case Order::ROUND_TOTAL:
                 $round_type = 'total';
-            break;
-        case Order::ROUND_LINE;
+                break;
+            case Order::ROUND_LINE:
```

Comment:
Somone typed semi colon here, they probably meant regular colon? I'm sorry I mixed this in, probably should've made a separate PR